ivadomed/scripts/curate_data_demo.py

```diff
@@ -0,0 +1,224 @@
+"""
+The data are structure in a folder such as:
+dataset/
+├── 20220309_demo_mouse_07_0001
+│         ├── image.png
+│         ├── mask.png
+│         ├── mask_seg-axon-manual.png
+│         ├── mask_seg-myelin-manual.png
+│         └── pixel_size_in_micrometer.txt
+├── 20220309_demo_mouse_07_0002
+│         ├── image.png
+│         ├── mask.png
+│         ├── mask_seg-axon-manual.png
+│         ├── mask_seg-myelin-manual.png
+│         └── pixel_size_in_micrometer.txt
+└── 20220309_demo_mouse_07_0003
+          ├── image.png
+          ├── mask.png
+          ├── mask_seg-axon-manual.png
+          ├── mask_seg-myelin-manual.png
+          └── pixel_size_in_micrometer.txt
+
+"""
+
+# Define various dependencies
+
+import glob
+import os
+import shutil
+import json
+import argparse
+import subprocess
+import csv
+from textwrap import dedent
+
+## Dictionary for images
+images = {
+    "image.png": "_TEM.png"
+}
+
+## Dictionary for derivatives
+der = {
+    "mask.png": "_TEM_seg-axonmyelin-manual.png",
+    "mask_seg-axon-manual.png": "_TEM_seg-axon-manual.png",
+    "mask_seg-myelin-manual.png": "_TEM_seg-myelin-manual.png"
```

Comment:
Here the keys of the dict would need to be changed if we choose to change the name of the demo files.